<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SAKHidDeviceController</class>
 <widget class="QWidget" name="SAKHidDeviceController">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>219</width>
    <height>82</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="3" column="0" colspan="2">
    <widget class="QCheckBox" name="noMouseCheckBox">
     <property name="text">
      <string>不显示鼠标设备</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QCheckBox" name="noKeyboardCheckBox">
     <property name="text">
      <string>不显示键盘设备</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>选择HID设备</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QComboBox" name="hidDeviceComboBox"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
